sanic/app.py

```diff
@@ -946,7 +960,7 @@ async def handle_request(self, request: Request):  # no cov
                 ...
                 await response.send(end_stream=True)
             elif isinstance(response, ResponseStream):
-                resp = await response(request)
+                resp = await response(request)  # type: ignore
```

Comment:
What are all these `# type: ignore` silencing?

Reply:
`ReportStream`. I want to get rid of it, so I was not overly concerned. But you are right. Will fix properly.